src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2274,6 +2286,49 @@ export class StudyViewPageStore {
         default: []
     });
 
+    readonly oncokbCancerGenes = remoteData<CancerGene[]>({
+        await: () => [],
+        invoke: async () => {
+            return oncoKBClient.utilsCancerGeneListGetUsingGET({});
+        },
+        onError: (error => {
+            console.log(error);
+        }),
+        default: []
+    });
+
+    readonly oncokbGenes = remoteData<OncokbGene[]>({
+        await: () => [],
+        invoke: async () => {
+            return oncoKBClient.genesGetUsingGET({});
+        },
+        onError: (error => {
+            console.log(error);
+        }),
+        default: []
+    });
+
+    @computed
+    get oncokbCancerGeneEntrezGeneIds() {
```

Comment:
As @alisman mentioned, we've run into issues with this a few times, and the conclusion we've come to is that,  in any case where a @computed can possibly not yet be ready (e.g. oncokbAnnotatedGeneEntrezGeneIds, since below you're waiting for oncokbCancerGenes before using oncokbAnnotatedGeneEntrezGeneIds), that it should just be made into a mobxpromise so it can be properly awaited and so that it models the true nature of that data as being unready then being ready.